### PR TITLE
Restyle

### PR DESCRIPTION
**Pull request by tyarkoni**
Updated the asset libraries for more recent datatables and bootstrap libraries, and cleaned up the appearance a bit. There are probably some minor quirks to clean up before merging, but take a look and let me know if you have comments/fixes.

### CODE REVIEW

**Comment by infocortex** [comment]
good idea, I did notice that the datatables are not loading the default tab on `collection_details` and `image_details`, and the js events are not binding in generate on `collection_details`.  (the upload folder and upload  zip buttons don't work atm).

last time this was caused by the incorrect datatable columns, fixed in 66e8853, but I don't think thats the case here.

**Comment by infocortex** [comment]
another very minor thing, the main navigation bar is in a full 12-column row, but most of the layouts are in a 10-column, so the margins can be a little visually inconsistent on some templates.

**Comment by tyarkoni** [comment]
Thanks for the feedback--I was kind of torn between making everything 12 columns for consistency, vs. doing some 10 and some 12. Maybe 10 throughout would make more sense... I'll try that and see how it looks.

I'll take a look at the datatables issue, I'm guessing there are probably some variables I forgot to rename (the newer version completely breaks the old API).

**Comment by chrisfilo** [comment]
@tyarkoni we had some mayor changes - could your rebase?

**Comment by tyarkoni** [comment]
Sorry, I guess I missed this comment a few months ago. I won't be able to work on this for the next few weeks though, so don't wait on me if someone else has time to revisit this.

File: neurovault/apps/main/static/css/datatables-bootstrap.css [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+
+/*
+ * FixedColumns styles
+ */
+div.DTFC_LeftHeadWrapper table,
+div.DTFC_LeftFootWrapper table,
+div.DTFC_RightHeadWrapper table,
+div.DTFC_RightFootWrapper table,
+table.DTFC_Cloned tr.even {
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/datatables-bootstrap.css]: Codacy Issue found: [Element (table.DTFC_Cloned) is overqualified, just use .DTFC_Cloned without element name.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/826454937/issues/source?bid=2767031&fileBranchId=2767091#l266)

File: neurovault/apps/main/static/css/datatables-bootstrap.css [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+	line-height: 28px;
+	margin: 1em;
+}
+
+div.DTTT_print_info p {
+	font-size: 14px;
+	line-height: 20px;
+}
+
+div.dataTables_processing {
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/datatables-bootstrap.css]: Codacy Issue found: [Element (div.dataTables_processing) is overqualified, just use .dataTables_processing without element name.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/826454937/issues/source?bid=2767031&fileBranchId=2767091#l236)

File: neurovault/apps/main/static/css/datatables-bootstrap.css [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+	-moz-box-sizing: content-box;
+	box-sizing: content-box;
+}
+
+
+table.dataTable {
+	clear: both;
+	margin-top: 6px !important;
+	margin-bottom: 6px !important;
+	max-width: none !important;
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/datatables-bootstrap.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/826454937/issues/source?bid=2767031&fileBranchId=2767091#l65)

File: neurovault/apps/main/static/css/datatables-bootstrap.css [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+
+div.dataTables_processing {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 100%;
+    height: 60px;
+    margin-left: -50%;
+    margin-top: -25px;
+    padding-top: 20px;
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/datatables-bootstrap.css]: Codacy Issue found: [Using height with padding-top can sometimes make elements larger than you expect.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/826454937/issues/source?bid=2767031&fileBranchId=2767091#l244)

File: neurovault/apps/main/static/css/datatables-bootstrap.css [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+div.dataTables_scrollHead table thead tr:last-child th:first-child,
+div.dataTables_scrollHead table thead tr:last-child td:first-child {
+	border-bottom-left-radius: 0 !important;
+	border-bottom-right-radius: 0 !important;
+}
+
+div.dataTables_scrollBody table {
+	border-top: none;
+	margin-top: 0 !important;
+	margin-bottom: 0 !important;
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/datatables-bootstrap.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/826454937/issues/source?bid=2767031&fileBranchId=2767091#l108)

File: neurovault/apps/main/static/css/datatables-bootstrap.css [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+}
+
+div.dataTables_scrollBody tbody tr:first-child th,
+div.dataTables_scrollBody tbody tr:first-child td {
+	border-top: none;
+}
+
+div.dataTables_scrollFoot table {
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/datatables-bootstrap.css]: Codacy Issue found: [Element (div.dataTables_scrollFoot) is overqualified, just use .dataTables_scrollFoot without element name.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/826454937/issues/source?bid=2767031&fileBranchId=2767091#l116)

File: neurovault/apps/main/static/css/datatables-bootstrap.css [hunk 7]
```diff
@@ -0,0 +1,313 @@
...
+
+div.dataTables_scrollHead table thead tr:last-child th:first-child,
+div.dataTables_scrollHead table thead tr:last-child td:first-child {
+	border-bottom-left-radius: 0 !important;
+	border-bottom-right-radius: 0 !important;
+}
+
+div.dataTables_scrollBody table {
+	border-top: none;
+	margin-top: 0 !important;
```
Comment (chrisfilo) [inline neurovault/apps/main/static/css/datatables-bootstrap.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/826454937/issues/source?bid=2767031&fileBranchId=2767091#l107)

**Comment by chrisfilo** [comment]
This has been addressed by @vsoch

**Pull request closed without merging**